src/api/spec/controllers/webui/cloud/ec2/upload_jobs_controller_spec.rb

```diff
@@ -0,0 +1,179 @@
+require 'rails_helper'
```

Comment:
Maybe combine this commit with the commit which does the splitting.

Reply:
Done